fgpyo/collections/__init__.py

```diff
@@ -1,38 +1,38 @@
 """
-Functions for Working with Collections
---------------------------------------
+# Functions for Working with Collections
 
 This module contains classes and functions for working with collections and iterators.
 
-Examples of a "Peekable" Iterator
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+## Examples of a "Peekable" Iterator
 
 "Peekable" iterators are useful to "peek" at the next item in an iterator without consuming it.
 For example, this is useful when consuming items in iterator while a predicate is true, and not
 consuming the first element where the element is not true.  See the
-:func:`~fgpyo.collections.PeekableIterator.takewhile` and
-:func:`~fgpyo.collections.PeekableIterator.dropwhile` methods.
+[`takewhile()`](index.md#fgpyo.collections.PeekableIterator.takewhile) and
+[`dropwhile()`](index.md#fgpyo.collections.PeekableIterator.dropwhile) methods.
```

Comment:
```suggestion
[`takewhile()`](#fgpyo.collections.PeekableIterator.takewhile) and
[`dropwhile()`](#fgpyo.collections.PeekableIterator.dropwhile) methods.
```

I think this will work and is more clear (IMHO) that it's an anchor within the same page.